discovery/pkg/openstack/translate.go

```diff
@@ -104,13 +103,10 @@ func loadbalancerLabels(lb loadbalancers.LoadBalancer) map[string]string {
 	}
 }
 
-// the service name in openstack is obtained from the LB's name and ID. If the
-// name is empty, the service name is the LB's ID.
+// use the load balancer ID as the service name
+// context: heptio/gimbal #216
 func serviceName(lb loadbalancers.LoadBalancer) string {
 	lbName := lb.ID
-	if lb.Name != "" {
-		lbName = fmt.Sprintf("%s-%s", lb.Name, lb.ID)
-	}
 	return strings.ToLower(lbName)
```

Comment:
Is the ID ever upper case to need the `ToLower`? Also, maybe could just simplify this to return: `strings.ToLower(lb.ID)`

Reply:
I have seen uppercase UUIDs in the past, so left the ToLower in there for good measure.

I'll simplify the code down to that single line.